index.js

```diff
@@ -7,5 +7,11 @@ if (major >= 5) {
   base = './src/';
 }
 
-module.exports = require(base + 'soap');
+module.exports = {
+  'soapModule': require(base + 'soap'),
+  'httpModule': require(base + 'http'),
```

Comment:
I don't see Node.js folks use the `Module` suffix. It can be just `soap` and `http`.